### PR TITLE
Fix input values for 64-bit integer support

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #13906)

### CODE REVIEW

File: test/tests/samplers/base/tests [hunk 1]
```diff
@@ -151,15 +151,15 @@
...
+      cli_args = "Samplers/sample/num_cols=1000000000 UserObjects/test/test_type=getLocalSamples"
 
       detail = "if the number of entries in the local sample matrix exceeds the maximum allowed, and"
     []
     [getNextLocalRow_max]
       type = RunException
       input = errors.i
       expect_err = "The number of entries in the std::vector \(1000000000\) exceeds the allowed limit of 429496729."
-      cli_args = "Samplers/sample/num_cols=1e9 UserObjects/test/test_type=getNextLocalRow"
+      cli_args = "Samplers/sample/num_cols=1000000000 UserObjects/test/test_type=getNextLocalRow"
```
Comment (permcody) [inline test/tests/samplers/base/tests]: Does 1e9 not work?

File: test/tests/samplers/distribute/tests [hunk 2]
```diff
@@ -9,6 +9,7 @@
       command = 'execute.py'
       check_files = 'distribute_none.csv distribute_on.csv distribute_off.csv'
       recover = false
+      valigrind = NONE
```
Comment (permcody) [inline test/tests/samplers/distribute/tests]: Valigrind huh? Is that some fancy version of valgrind? 😉

**Review by permcody — CHANGES_REQUESTED** [review]

File: test/tests/samplers/base/tests (hunk 1 above)
Comment (aeslaughter) [reply test/tests/samplers/base/tests]: It doesn't seem too, this is an exploratory change.

Comment (aeslaughter) [reply test/tests/samplers/base/tests]:
```
field node 'Samplers/sample/num_cols' does not hold an int-typed value (val='1e9')
```

**Review by permcody — APPROVED** [review]

**Pull request merged**